app/cisc/cisc.go

```diff
@@ -290,8 +324,66 @@ func sshSync(c *cli.Context) error {
 	log.Fatal("Not yet implemented")
 	return nil
 }
-func proposeSendVoteUpdate(cfg *identity.Identity, p *identity.Config) {
-	log.ErrFatal(cfg.ProposeSend(p))
-	log.ErrFatal(cfg.ProposeVote(true))
-	log.ErrFatal(cfg.ConfigUpdate())
+
+func followAdd(c *cli.Context) error {
+	if c.NArg() < 2 {
+		log.Fatal("Please give a group-definition, an ID, and optionally a service-name of the skipchain to follow")
+	}
+	cfg, _ := loadConfig(c)
+	group := getGroup(c)
+	idBytes, err := hex.DecodeString(c.Args().Get(1))
+	log.ErrFatal(err)
+	id := identity.ID(idBytes)
+	newID, err := identity.NewIdentityFromCothority(group.Roster, id)
+	log.ErrFatal(err)
+	if c.NArg() == 3 {
+		newID.DeviceName = c.Args().Get(2)
+	} else {
+		var err error
+		newID.DeviceName, err = os.Hostname()
+		log.ErrFatal(err)
+	}
+	cfg.Follow = append(cfg.Follow, newID)
+	cfg.writeAuthorizedKeys(c)
```

Comment:
Is there a big warning somewhere saying that using cisc for ssh key mgmt will write into your authroized_key file ? If not, there should be in my opinion, because this is not something to take lightly: `Oh, I'll just add some keys to your authorized_key file, don't worry !`  I know it's not necessarily related to this PR but still..